src/material/tabs/paginated-tab-header.ts

```diff
@@ -118,15 +119,14 @@ export abstract class MatPaginatedTabHeader implements AfterContentChecked, Afte
    * Whether pagination should be disabled. This can be used to avoid unnecessary
    * layout recalculations if it's known that pagination won't be required.
    */
-  @Input()
-  disablePagination: boolean = false;
+  @Input() disablePagination: boolean = false;
 
   /** The index of the active tab. */
-  get selectedIndex(): number { return this._selectedIndex; }
+  @Input() get selectedIndex(): number { return this._selectedIndex; }
```

Comment:
This isn't supposed to be an input either, because it'll propagate to the tab nav bar which has a different way of setting which tab is selected (via the `active` property).